src/main/groovy/com/lesfurets/jenkins/unit/declarative/AgentDeclaration.groovy

```diff
@@ -55,6 +55,10 @@ class AgentDeclaration extends GenericPipelineDeclaration {
         this.docker = createComponent(DockerAgentDeclaration, closure)
     }
 
+    def kubernetes(Boolean _) {
```

Comment:
IMHO this should be `boolean`, not `Boolean`, unless there is a compelling reason to use a boxed type.

Reply:
yeah, no reason for `Boolean`. will update